app/views/categories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="card-columns">
+  <% @categories.each do |category| %>
+    <%= link_to category_path(category) do %>
+      <div class="container card-container">
+        <img src="<%= category.icon %>" class="card-image" alt="Category Icon">
+        <div class="card-contents">
+          <h5 class="card-name"><%= category.name %></h5>
+          <p class="card-date"><%= category.created_at.strftime('%d %B, %Y') %></p>
+        </div>
+        <p class="total-cash">$<%= @category_totals[category.id] %>
+        </p>
+      </div>
+    <% end %>
+  <% end %>
+  <div class="container add-category">
+    <%= link_to "Add New Category", new_category_path, class: "btn btn-default" %>
+  </div>
```

Comment:
- [ ] Just as suggested by the previous reviewer, kindly ensure that the big green button for adding a category is fixed at the bottom of the display regardless of whether the page is empty or not. This will help us get a design consistent with the design template.
### Design Guideline
![image](https://github.com/ismailmunyentwari9/transact_track/assets/46839548/2c4ad6ce-0495-4543-b7f8-0e7118726786)

### Your Implementation
![image](https://github.com/ismailmunyentwari9/transact_track/assets/46839548/cb9a45ba-be7d-4dec-baa3-4f0c24bbbfff)

- [ ] [OPTIONAL] The button text is difficult to read at the moment due to your use of colors. Please consider using the exact text color used in the template design in order to improve accessibilty 👍